src/features/movies/MovieDetails/Backdrop/styled.js

```diff
@@ -0,0 +1,52 @@
+import styled from "styled-components";
+
+
+export const BackdropWrapper = styled.div`
+  justify-content: center;
+  animation: eclipse 0.75s ease-in-out;
+  animation-fill-mode: forwards;
+  `
+
+export const Wrapper = styled.section`
+max-width: 1400px;
+margin: 0 auto;
+padding: 0 16px;
+position: relative;
+`;
+
+export const BackdropBody = styled.div`
+  position: relative;
+  padding: 0 16px;
+  aspect-ratio: 16 / 9;
+  background-image: ${({ background }) => `url(${background})`};
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  animation: reveal 2s ease-in-out 0.75s;
+  animation-fill-mode: forwards;
+  opacity: 0;
+  `;
+
+export const Title = styled.h1`
+color: ${({ theme }) => theme.color.white};
```

Comment:
@Dawid-Zieba you missed "S" in this code, correct it to: `color: ${({theme }) => theme.colors.white};`